polaris/hub/client.py

```diff
@@ -404,8 +404,6 @@ def open_zarr_file(
         except HTTPStatusError as error:
             # In this case, we can pass the response to provide more information
             raise PolarisHubError(message="Error opening Zarr store", response=error.response) from error
-        except Exception as error:
-            raise PolarisHubError(message="Error opening Zarr store") from error
```

Comment:
**Question**
Was this for debugging? If so, we might want to restore them, to wrap underlying errors into something more readable.

Reply:
There is another `except` right before this one. I thought it was an unintended duplicate.  
So it is only for readability of the error? I will add a comment for it.

Reply:
They're both useful. The first block catches errors of type `HTTPStatusError` and passes additional information available in this type to `PolarisHubError` (e.g. `response=error.response`) for display/formatting. The second block catches all other types of exceptions. 

We can't use a single block because `HTTPStatusError` contains additional error info that `Exception` does not.